README.md

```diff
@@ -1,2 +1,53 @@
 # MoSDeF-dihedral-fit
-MoSDeF-dihedral-fit: A simple software package to fit dihedrals via the MoSDeF software
+MoSDeF-dihedral-fit: A simple software package to fit dihedrals via the MoSDeF software.
+
+This package is a tool to add dihedral parameters for [Ryckaert-Bellemans](https://manual.gromacs.org/current/reference-manual/functions/bonded-interactions.html#proper-dihedrals-ryckaert-bellemans-function) dihedral based forcefields by fitting QM calculations for the rotation about the dihedral angle to this functional form. The main functionality is found through the following code snippet.
+```python
+import unyt as u
+from mosdef_dihedral_fit.dihedral_fit.fit_dihedral_with_gomc import fit_dihedral_with_gomc
+fit_dihedral_with_gomc(
+        ["HC", "CT", "CT", "HC"], #atomclass names of the dihedral
+        "molecule.mol2" # mol2 file with relevant structure
+        "compound.xml", # xml file with other atomtyped parameters in foyer format
+        298.15 * u.Kelvin, # relevant temperature
+        gomc_binary_directory, # path to binary command from GOMC install
+        {"HC_CT_CT_HC_multiplicity_1.log": []}, # log file to store info
+        zeroed_dihedral_atom_types=None,
+        qm_engine="gaussian",
+        VDWGeometricSigma=False,
+        atom_type_naming_style="general",
+        gomc_cpu_cores=1,
+        fit_min_validated_r_squared=0.99,
+        fit_validation_r_squared_rtol=1e-03,
+    )
+import os
+os.system("cat RB_torsion_k_constants_fit_energy.txt")
+```
+
+## Installation/Setup
+```
+conda install -c conda-forge mosdef_dihedral_fit
+git clone https://github.com/GOMC-WSU/GOMC.git
+cd GOMC
+chmod u+x metamake.sh
+./metamake.sh
+./GOMC_<CPU|GPU>_XXXX +p4 in.conf # set to 4 threads
```

Comment:
GOMC needs to be installed. Testing your install is certainly best practice. If something is screwed up in the install, it would be good to identify that that was the issue and not some bug in `mosdef-dihedral-fit` package.

Reply:
Yeah, I was not aware GOMC Chas it’s own tests.

Reply:
I would write like remove this as they will not run this command as they conf have test files for GOMC 

./GOMC_<CPU|GPU>_XXXX +p4 in.conf

Reply:
Instead of running GOMC text, you can provide a link to the mosdef comic examples to test the GOMC install, which has video tutorials (https://github.com/GOMC-WSU/GOMC_Examples/tree/main/MoSDef-GOMC/Part_5_NPT_Example), if you think it is important